src/test/java/org/calrissian/flowmix/api/aggregator/CountAggregatorTest.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2015 Calrissian.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.calrissian.flowmix.api.aggregator;
+
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+/**
+ * @author Miguel A. Fuentes Buchholtz
```

Comment:
Oh no! This is the same as the deprecated thing, it's just an internal company thing but I dont care about my specific name in OS :)

I use sonarqube to find weird things in code using a custom rule